source/common/runtime/runtime_features.cc

```diff
@@ -172,6 +172,8 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_use_network_type_socket_option);
 // https://github.com/envoyproxy/envoy/blob/main/api/envoy/extensions/transport_sockets/tls/v3/tls.proto#L29.
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_quic_disable_client_early_data);
 
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_ext_proc_graceful_grpc_close);
```

Comment:
Can we set this runtime flag to TRUE by default so the fix can be tested?

Reply:
I will test it internally first before changing the flag to TRUE.